src/navigation/wallet/screens/JoinMultisig.tsx

```diff
@@ -153,28 +123,64 @@ const JoinMultisig = () => {
           }),
         )) as Wallet;
 
-        navigation.dispatch(
-          CommonActions.reset({
-            index: 2,
-            routes: [
-              {
-                name: 'Tabs',
-                params: {screen: 'Home'},
-              },
-              {
-                name: 'Wallet',
-                params: {screen: 'KeyOverview', params: {id: key.id}},
-              },
-              {
-                name: 'Wallet',
-                params: {
-                  screen: 'WalletDetails',
-                  params: {walletId: wallet.id, key},
-                },
-              },
-            ],
-          }),
-        );
+        // TODO
+        wallet.getStatus({network: 'livenet'}, (err: any, status: Status) => {
+          if (err) {
+            // TODO
+            console.log(err);
+          }
+
+          if (status.wallet && status.wallet.status === 'complete') {
+            wallet.openWallet({}, () => {
+              navigation.dispatch(
+                CommonActions.reset({
+                  index: 2,
+                  routes: [
+                    {
+                      name: 'Tabs',
+                      params: {screen: 'Home'},
+                    },
+                    {
+                      name: 'Wallet',
+                      params: {screen: 'KeyOverview', params: {id: key.id}},
+                    },
+                    {
+                      name: 'Wallet',
+                      params: {
+                        screen: 'WalletDetails',
+                        params: {walletId: wallet.id, key},
+                      },
+                    },
+                  ],
+                }),
+              );
+            });
+          } else {
+            navigation.dispatch(
+              CommonActions.reset({
```

Comment:
question: else will execute even when getStatus return err. is it intensional?

Reply:
Let's ignore the error in this case and just navigate to the key view